test/assert.js

```diff
@@ -30,6 +30,8 @@ test('.ok()', function (t) {
 		assert.ok(true);
 	});
 
+	t.same(assert.ok, assert.truthy);
```

Comment:
Just noticed that even in this very PR it is evident that I confused `.same()` with `.is()` :-)